easy-ui-react/src/VerticalStack/VerticalStack.tsx

```diff
@@ -39,15 +36,24 @@ export type VerticalStackProps = {
   /** Vertical alignment of children */
   align?: Align;
 
-  /** Horizontal alignment of children */
-  inlineAlign?: InlineAlign;
+  /** Contents of the vertical stack. */
+  children: React.ReactNode;
+
+  /** Custom className for the vertical stack. */
+  className?: string;
```

Comment:
**question** so i see the use for this and its sensible, but do you have any concerns about having this exposed and this also being a component consumers can use?

Reply:
i do have concerns, yes. i think you're right. i removed it and modified the component structure